### PR TITLE
Initial commit of user interface or front end

### PR DESCRIPTION
**Pull request by hfakhreddine86**
Please take a look at the naming of tags, Id's, and classes. If they need to be changed please review and comment. If names need to be changed, we will also have to go into the CSS and name them accordingly.

### CODE REVIEW

File: assets/script/script.js [hunk 1]
```diff
@@ -1,17 +1,17 @@
 // Test code for the functioning of the Server
 
+$(document).ready(function () {
```
Comment (abonnema) [inline assets/script/script.js]: This is still the same command

File: assets/css/style.css [hunk 2]
```diff
@@ -19,23 +23,86 @@ body .navbar-fixed {
...
+    background: rgb(53, 159, 36);
+    color: black;
+}
+
+.style {
+    color: rgb(58, 58, 58) !important;
+    font-size: 24px !important;
+}
+
+[type="radio"]:checked+label:after,
```
Comment (abonnema) [inline assets/css/style.css]: Good Radio labels, for the future, somehow  if radio === checked, { (make the search results only appear on Vegan/Vegetarian)...

File: assets/script/script.js (hunk 1 above)
Comment (nscox) [reply assets/script/script.js]: I took a look see and its all dope!

File: index.html [hunk 3]
```diff
@@ -12,11 +12,15 @@
     <link type="text/css" rel="stylesheet" href="assets/css/materialize.min.css" media="screen,projection" />
     <link href="https://fonts.googleapis.com/css?family=Permanent+Marker" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css?family=Exo+2" rel="stylesheet">
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+    <script type="text/javascript" src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
```
Comment (abonnema) [inline index.html]: We need to add a Google API AJAX script line in the future...

**Pull request merged**